bench-run/publication/metrics.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
```

Comment:
Please add an empty line between shebang and import.

Reply:
fixed